src/helpers/pathCreator.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+function globalizationDestination(outDestination, topic, regionCode){
+  return function(version, isInternal){
+    const internalPath = isInternal ? '/internal' : '';
+    const regionPath = regionCode ? `/${regionCode}` : '';
+    const shortName = (isInternal && topic.shortNameInternal) ? topic.shortNameInternal : topic.shortName;
+    return `${outDestination}${internalPath}/${topic.type}${regionPath}/${shortName}/${version}`;
```

Comment:
Oh my God 😮 
Maybe there is some other attribute available in config? :D
Maybe there is field you could do like: `url.replace('services/', 'services/' + code + '/')` or sth like that